pyEpiabm/pyEpiabm/cell.py

```diff
@@ -0,0 +1,30 @@
+from .microcell import Microcell
+
+
+class Cell:
+    """Class representing a Cell (Subset of Population)
+    Collection of :class:`Microcell` s and :class:`Person` s
+    """
+    def __init__(self):
+        """Constructor Method
```

Comment:
Not sure why we should need a doctoring for the ``__init__`` method:)), but don't forget the `.` at the end of each docstring

Reply:
I think PEP8 says that all public methods (including __init__()) should have docstrings - will make sure they all have full stops at the end though!